frontend/src/HomePage/HomePage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import axios from 'axios';
+import NavBar from '../Navbar/NavBar';
+
+class HomePage extends React.Component {
+	state = { details: [] };
+
+	componentDidMount() {
+		//let data;
+		axios
+			.get('http://localhost:8000/api/posts/')
+			.then((res) => this.setState({ details: res.data }))
+			.catch((err) => console.log(err));
+	}
```

Comment:
Use react hooks, this is old way.

But first, create .env file in your FE which has a value of this:
```
REACT_APP_SERVER_API=http://localhost:8000/api
```

Then create a base api, so you want add the base url again:
```
import axios from 'axios';

const API = axios.create({
  baseURL: `${process.env.REACT_APP_SERVER_API}/api`
});

export default API;
```

Lastly, import your base api then use react hook useEffect instead of componentDidMount
```suggestion
//import base axios API here...

	useEffect(() => {
	
		API
			.get('/posts/')
			.then((res) => setPosts(res.data))
			.catch((err) => console.log(err));
	}, [ ])
```

Reply:
Should I make folder for this one po or just directly in FE folder?